fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -430,14 +456,54 @@ case $DistortionCorrection in
                 --smoothingfwhm="2" \
                 --inputdir="$WD"
         fi
-            
+
 
         ;;
 
+    ${NONE_METHOD_OPT})
+
+            # NOTE: To work with later code a number of fake files are created 
+            #       with names that do not reflect the true nature of the file
+            #       this is to be considered an initial fix for datasets that 
+            #       do not include data that enable distortion correction and
+            #       will be replaced with a cleaner code. 
```

Comment:
If if don't create the proposed "cleaner code" now, it is very unlikely to get done in the future.  I'd still like to see that get done in this PR.  Or, is this comment no longer entirely accurate...?

Reply:
Indeed the code has been cleaned. The only "Fake" image is the uniform Jacobian that is created here. I have changed the note to:

```
# NOTE: To work with later code a uniform Jacobian is created.
```